pkg/process/net/common_unsupported.go

```diff
@@ -5,13 +5,14 @@ package net
 import (
 	model "github.com/DataDog/agent-payload/process"
 	"github.com/DataDog/datadog-agent/pkg/ebpf"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
 )
 
 // RemoteSysProbeUtil is not supported
 type RemoteSysProbeUtil struct{}
 
-// SetSystemProbeSocketPath is not supported
-func SetSystemProbeSocketPath(_ string) {
+// SetSystemProbePath is not supported
+func SetSystemProbePath(_ *config.AgentConfig) {
```

Comment:
if it's not supported should it return an error?

Reply:
For now, I think we're leaving it without an error since this function is only ever called before GetRemoteSystemProbeUtil, which will return an error if the probe path is not set (it's unsupported).